### PR TITLE
Changed CI to run oneDPL examples instead of oneAPI

### PR DESCRIPTION
**Pull request by dparanic**

### CODE REVIEW

File: jenkinsfiles/RHEL_Check.groovy [hunk 1]
```diff
@@ -285,29 +283,40 @@ pipeline {
...
+                                        def convex_hull_return_value = runExample("convex_hull")
+                                        def dot_product_return_value = runExample("dot_product")
+                                        def histogram_return_value = runExample("histogram")
+                                        def random_return_value = runExample("random")
+
+                                        def test_pass_status = true
+                                        if (gamma_return_value != 0) {
+                                            echo "gamma-correction check failed."
+                                            test_pass_status = false
+                                        }
```
Comment (AlexVeprev) [inline jenkinsfiles/RHEL_Check.groovy:297]: Is it possible to "pack" these parts into `runExample` function as well? Similar to Windows.

Comment (dparanic) [reply jenkinsfiles/RHEL_Check.groovy]: Done

File: examples/convex_hull/src/main.cpp [hunk 2]
```diff
@@ -14,7 +14,7 @@
 #include <fstream>
 #include <vector>
 
-#include "utils.h"
+#include "utils.hpp"
```
Comment (capatober) [inline examples/convex_hull/src/main.cpp:17]: It's covered by https://github.com/oneapi-src/oneDPL/pull/170/files#diff-324573e248045a8462e86fdd8ae69d302e832b699e32fdd201c67d4205973aa8R17 
You don't need to do it here

**Review by AlexVeprev — APPROVED** [review]

File: examples/convex_hull/src/main.cpp (hunk 2 above)
Comment (dparanic) [reply examples/convex_hull/src/main.cpp]: Rolled back.

**Pull request merged**